docs/creating-a-player/xmds.md

```diff
@@ -621,7 +666,41 @@ or it can provide more information
  - method (optional): The method.
  - thread (optional): The Thread that the log message executed on.
 
+#### Display Alerts
+**Since CMS 4.1 and later** displays have been able to submit alerts to the CMS via the SubmitLog method. Alerts represent interesting events occurring on the media player, 
+such as the network going offline, application restarts, memory alerts, etc.
+
+These are submitted by setting the category attribute to `category="alert"`.
 
+The following format is expected for alerts:
+
+```xml
+<logs>
+    <log date="Y-m-d H:i:s" category="alert">
```

Comment:
the category is `event` not `alert` - although to be fair alert might make more sense, but we would need to change the CMS code.

Reply:
Ah OK, I'll change it.